lib/index.coffee

```diff
@@ -55,4 +55,38 @@ class DockerGC
 			console.log('GC: Done.')
 			return _.every(results)
 
+	getDaemonFreeSpace: () ->
+		# Ensure the image is available (if it is this is essentially a no-op)
+		@docker.pull('alpine')
+		.then (stream) ->
+			new Promise (resolve, reject) ->
+				stream.resume()
+				stream.once('error', reject)
+				stream.once('end', resolve)
+		.then =>
+			@docker.run 'alpine',
+				[
+					'/bin/sh',
+					'-c',
+					"df -k / | tail -n +2 | awk '{ print $3,$4 \"\\t\" }'"
+				]
+			.then (container) ->
+				container.logs(stdout: 1)
+			.then (logs) ->
+				new Promise (resolve, reject) ->
+					logStr = ''
+					logs
+					.on('data', (data) -> logStr += data.toString('utf-8'))
+					.on('end', () -> resolve(logStr))
+					.on('error', reject)
+			.then (spaceStr) ->
+				parts = spaceStr.split(/\s+/)
+				used = parseInt(parts[0])
+				total = parseInt(parts[1])
```

Comment:
isn't this the available space?
df returns `total used available`

Reply:
Also, why mixing tail, awk and js for extracting  it ?

Reply:
I wanted the output of the container to be the most basic it can be. I'm not against just parsing it in JS, I was initially going to do it a different way (have a specially created image) so it would have been required like this, but it can be done either way now.

Reply:
You could remove tail and awk and use
```coffeescript
[dev, total, used, free] = a.trim().split('/\n')[1].split(/\s+/)
alert(total)
alert(used)
alert(free)
```